src/types/builtIns.ts

```diff
@@ -14,7 +14,11 @@ const types: Array<IDocTypes> = [
         name: 'url',
         unions: [],
         intersections: [],
-        definition: {},
+        definition: {
+          socket: EDocParamType.Hash,
+          path: EDocParamType.Hash,
+          fragment: EDocParamType.Hash,
```

Comment:
Should this be `hash | unknown`? I don't recall if the generator has a way to represent unknown vs any at the moment.

Reply:
Yeah, it's probably fine to leave as just `hash`. The `any` keyword would override the union.